mutmut/__main__.py

```diff
@@ -181,7 +178,7 @@ def main(command, argument, argument2, paths_to_mutate, disable_mutation_types,
         mutation_types_to_apply = set(mutations_by_type.keys())
         invalid_types = None
     if invalid_types:
-        raise click.BadArgumentUsage(f"The following are not valid mutation types: {', '.join(invalid_types)}. Valid mutation types are: {', '.join(mutations_by_type.keys())}")
+        raise click.BadArgumentUsage(f"The following are not valid mutation types: {', '.join(sorted(invalid_types))}. Valid mutation types are: {', '.join(mutations_by_type.keys())}")
```

Comment:
Sorting the set creates deterministic output and therefore eliminates the flaky test.